spec/adhearsion/call_controller/dial_spec.rb

```diff
@@ -1485,6 +1485,17 @@ def dial_in_thread
             end
           end
         end
+
+        context 'when given a block with one argument' do
+          it "yields a block on the dial obj" do
+            expect(OutboundCall).to receive(:new).and_return other_mock_call
+            expect(other_mock_call).to receive(:dial).with(to, options).once
+            Thread.new do
+              expect {|b| subject.dial(to, options, &b)}.to yield_with_args Dial
+            end
+            sleep 0.1
```

Comment:
I suspect this `sleep` will cause problems on Travis as it's not always that fast. I think you should use a latch like the other examples.